exercises/complex-numbers/canonical-data.json

```diff
@@ -323,6 +323,14 @@
             "z": [1, 0]
           },
           "expected": ["e", 0]
+        },
+        {
+          "description": "Exponential of a number with real and imaginary part",
+          "property": "exp",
+          "input": {
+            "z": ["ln(2)", "pi"]
+          },
+          "expected": [2, 0]
```

Comment:
According to my calculations, and also according to your calculations in https://github.com/exercism/problem-specifications/pull/1095#issue-288173324, the answer is `-2`. But given that I see here `[2, 0]` that would be lead me to believe the answer is 2.

* Should this be `[-2, 0]` instead?
* Did we both  mess up in our calculations?

Reply:
> Should this be [2, 0] instead?

@petertseng Yes; very good catch. I must've missed the hyphen when creating the test case. I will make the change.